### PR TITLE
Fixed Storage Identity SAS bug and Issue #7309

### LINKED ISSUE #7309: Storage: Enable UserDelegationKey + SAS tests for 2019-02-02
The GetPropertiesAsync_BlobIdentitySAS, GetPropertiesAsync_ContainerIdentitySAS, and GetPropertiesAsync_SnapshotIdentitySAS tests were disabled when upgrading to 2019-02-02.

### PR DESCRIPTION
**Pull request by seanmcc-msft**
Resolves #7309

### CODE REVIEW

**Review by amnguye — APPROVED** [review]

**Comment by weshaggard** [comment]
Hit https://github.com/Azure/azure-sdk-for-net/issues/7490.

**Comment by weshaggard** [comment]
Re-run failed with duplicate artifact issue so triggering the entire pipeline :(

/azp run net - storage - ci

File: sdk/storage/Azure.Storage.Common/src/Constants.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ internal static class Constants
         /// Gets the default service version to use when building shared access
         /// signatures.
         /// </summary>
-        public const string DefaultSasVersion = "2018-11-09";
+        public const string DefaultSasVersion = "2019-02-02";
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Common/src/Constants.cs]: That was it...?  Sigh.  Thanks for fixing this!

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Common/src/Constants.cs (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Common/src/Constants.cs]: Ya, and it actually broke Identity SAS.

**Pull request merged**